lightning/src/sign/mod.rs

```diff
@@ -996,12 +996,7 @@ pub trait ChangeDestinationSourceSync {
 }
 
 /// A wrapper around [`ChangeDestinationSource`] to allow for async calls.
-#[cfg(any(test, feature = "_test_utils"))]
 pub struct ChangeDestinationSourceSyncWrapper<T: Deref>(T)
```

Comment:
Do you still need this public when using `OutputSweeperSync` which does the wrapping for you?

Reply:
Hmm if we do this, isn't this going to risk blocking tokio?

Reply:
Looks good. This is what we always wanted, isn't it?